src/cmd/services/m3dbnode/config/cache.go

```diff
@@ -38,6 +47,20 @@ func (c CacheConfigurations) SeriesConfiguration() SeriesCacheConfiguration {
 	return *c.Series
 }
 
+// PostingsListConfiguration returns the postings list cache configuration
+// or default if none is specified.
+func (c CacheConfigurations) PostingsListConfiguration() PostingsListCacheConfiguration {
+	if c.PostingsList == nil {
+		return PostingsListCacheConfiguration{
+			Size:        &defaultPostingsListCacheSize,
+			CacheRegexp: &defaultPostingsListCacheRegexp,
+			CacheTerms:  &defaultPostingsListCacheTerms,
```

Comment:
Why not just leave this all `nil` and let the `ThingOrDefault()` handle the case to provide the default value?

Reply:
sure